engine/installation/index.md

```diff
@@ -32,8 +32,16 @@ Docker on a different Linux distribution, consider
 
 ## On Cloud
 
-You can use Docker Cloud to automatically provision and manage your cloud
-instances:
+We provide official Docker solutions for running on AWS and Azure, which you can
+. You can read up on what Docker for AWS and Docker for Azure have to offer you
+[here](/docker-for-aws/why/) and [here](/docker-for-azure/why/) respectively.
+
+* [Docker for AWS](/docker-for-aws/)
+* [Docker for Azure](/docker-for-azure/)
+
+We also provide guides for connecting the following cloud providers to your
+Docker Cloud account to have Docker Cloud automatically provision and manage
```

Comment:
s/to have.../to automatically provision and manage your cloud instances using Docker Cloud.